test/behavior/slice.zig

```diff
@@ -166,3 +166,21 @@ test "slicing zero length array" {
     try expect(mem.eql(u8, s1, ""));
     try expect(mem.eql(u32, s2, &[_]u32{}));
 }
+
+test "slice sentinel access at comptime" {
```

Comment:
Looks like this test is not yet passing in stage2; if you move it to the slice_stage1.zig then you should be good to go.